src/Microsoft.ApplicationInsights/Extensibility/Implementation/External/MessageData_types.cs

```diff
@@ -54,6 +54,11 @@ internal partial class MessageData
         
         public IDictionary<string, string> properties { get; set; }
 
+        
+        
+        
+        public IDictionary<string, double> measurements { get; set; }
```

Comment:
Unrelated? (Was this just missing before?)

Reply:
yeah, I updated schemas in a batch. We didn't have measurements on logs before. But backend supports it and no reason to confuse anybody by not exposing it